Add queries to populate spatial_subset table with IMOS and CSIRO traject...

bpasquer opened the pull request:
...ory data

bpasquer committed:
Add queries to populate spatial_subset table with IMOS and CSIRO trajectory data

bpasquer committed:
New and amended scripts
-copied over xav's script to flag exact duplicates
- added query to subset WODB GLD and SUR data
- added  script to delete empty rows in WODB GLD dataset (rows with no Depth, Temp and PSal data)

bpasquer committed:
WODB : Add script for preprocessing of WODB csv files

bpasquer committed:
ATLAS: Trajectory data processing:
-created an index for  spatial_subset table in order to improve performance when running queries on large schemas
-created a table for aggregated Trajectory files containing info for identifying duplicate.
-deleted data from after 2015-01-01
-populated the aggregated Trajectory table

bpasquer committed:
ATLAS Trajectory processing: corrected quesy to spatially subset scehams

bpasquer committed:
Spatial subset query runs on data view instead of map view

bpasquer committed:
MARVL# Trajectorydata :
- improved query for populating trajectory data in spatial_subset
- improved query for flagging duplicates.

bpasquer committed:
Populate spatial_subset: modify queries according to Guillaumes comments

bpasquer committed:
Trajectory : add queries to remove replicate and other excuded data from schemas

ggalibert commented on sql_queries/PopulateSpatialSubset_TRAJECTORY.sql:
```diff
@@ -0,0 +1,410 @@
+
+---SOOP-TRV
+\echo 'SOOP-TRV'
+INSERT INTO marvl3.spatial_subset(
```
I've already merged in master a sql query to process SOOP TMV and TRV so that they are binned and sub-sampled over time. Unless you are doing something I am missing, you may want to delete this.

ggalibert commented on sql_queries/PopulateSpatialSubset_TRAJECTORY.sql:
```diff
@@ -0,0 +1,410 @@
+) -- Astrolabe SOOP-SST-NRT data processed to DM product only up to Apr 2013  
+---SOOP_TMV
+\echo 'SOOP_TMV'	
+INSERT INTO marvl3.spatial_subset(
```
Already addressed in master. Please compare and if all good delete this.

ggalibert commented on sql_queries/PopulateSpatialSubset_TRAJECTORY.sql:
```diff
@@ -0,0 +1,410 @@
+
+-- WODB UOR
+\echo 'WODB UOR'
+INSERT INTO marvl3.spatial_subset(
```
Already processed in master. Please compare and maybe delete your entry.

ggalibert commented on gradlew.bat:
```diff
@@ -1,90 +0,0 @@
-@if "%DEBUG%" == "" @echo off
```
Did you really need to delete this file?

bpasquer committed:
ATLAS - Trajectory Data:Deleted  WODB UOR SOOP-TMV and SOOP-TRV from query to populate source table

bpasquer committed:
ATLAS :Trajectory data:
-split the Populate Supatial_subest_Trajectory query into single query per table
-put back the gradlew.bat file previously deleted by mistake in a previous commit
-deletes redundant queries

bpasquer committed:
MARVL Trajectory data processing: Delete redundant queries

bpasquer committed:
Trajectory :Delete another unused query

ggalibert commented:
Is this still on the agenda? If so we need some help from a developer to resolve conflicts.

ggalibert commented:
@bpasquer I think you should close this PR if not relevant anymore.
